ZukoAzula2016/src/org/team1540/zukoazula/AutonomousModeForward.java

```diff
@@ -0,0 +1,25 @@
+package org.team1540.zukoazula;
+
+import ccre.channel.FloatInput;
+import ccre.instinct.AutonomousModeOverException;
+
+public class AutonomousModeForward extends AutonomousBase {
+
+    @Tunable(3.0f)
+    private FloatInput timeToDriveForward;
+    
+    @Tunable(0.5f)
+    private FloatInput drivingSpeed;
+    
+    public AutonomousModeForward() {
+        super("Drive Forward");
+    }
+
+    @Override
+    protected void runAutonomous() throws InterruptedException, AutonomousModeOverException {
+        if (timeToDriveForward.get() > 0.0f) {
+            float speed = Math.signum(drivingSpeed.get()) * Math.min(Math.abs(drivingSpeed.get()), 1.0f);
+            driveForTime((long)timeToDriveForward.get(), speed);
```

Comment:
When times are involved, `long`s and `int`s generally imply that the units are _not_ seconds, so this would actually be interpreting `timeToDriveForward` as _milliseconds_. 3 milliseconds isn't very long.